packages/RatPack.package/RPTests.class/instance/testCorrectContentDecoding.st

```diff
@@ -0,0 +1,12 @@
+test - decoding
+testCorrectContentDecoding
+	
+	RPTestHelper put: '/index' on: app headers: (Dictionary newFrom: {'Content-Type' -> 'test/contenttype'}) content: ''.
+	self assert: (app parsedRequestContents) equals: (RPMockDecoder decode: '').
```

Comment:
I think we should compare with a class method like `RPMockDecoder correctResponse` instead of decoding an empty string to make this more readable.